wfsim/core.py

```diff
@@ -278,7 +278,7 @@ def __call__(self, instruction):
             # shape of recarr is a bit strange
             instruction = np.array([instruction])
 
-        _, _, t, x, y, z, n_photons, recoil_type, *rest = [
+        _, _, t, _, x, y, z, n_photons, recoil_type, *rest = [
```

Comment:
Hey, would not it be easier to use here the array field-names? In this way you do not have to change always those lines when a new columns is added to the truth array. I know this will be conflicting with the pattern maps since the scipy.interpolated function cannot handle structured arrays for whatever reason, but you can circumvent this issue via: 

```
 pos = np.array([instructions['x'], 
                        instructions['y'], 
                        instructions['z']]).T
```

Reply:
Thanks, I'll need to check